app/src/main/java/com/codingblocks/cbonlineapp/ui/SkeletonCardUi.kt

```diff
@@ -17,11 +17,11 @@ import org.jetbrains.anko.topPadding
 import org.jetbrains.anko.view
 
 class SkeletonCardUi : AnkoComponent<ViewGroup> {
```

Comment:
remove this class it is not being used can't be used with skeleton layout

Reply:
Got it